nss-day-cohort-30/bangazon-site-mighty-mahi-mahi
# Cb ticket number21

Fixes:
- Navigation bar does not show View Cart, Sell Something, My Products, My Account, and Reports does not show if the user is not logged in
- My Products shows a status report for each of the users listed products

Changes proposed in this request & reasons behind organizational or architectural decisions:
- Created UserProductStatusModel to send as a viewmodel to the My Products view

Steps to test:
- Start Bangazon
-Make sure you have multiple products listed
-Click on My Products
-View the quantity available and number sold
-Create and complete a new order with at least one of all of your products
-Click on My Products and ensure the values have changed appropriately

System configuration (3rd party libraries to be installed, command line utilities to run, UI instructions, expected outcome):
-N/A

Commit message:
- View Cart, Sell Something, My Products, My Account, and Reports section hyperlinks only display in  the navigation bar if the user is logged in
- User's 'My Products' page displays the status of each of their listed products with the quantity available and the number sold
--

Link to feature ticket:
-https://github.com/nss-day-cohort-30/bangazon-site-mighty-mahi-mahi/issues/21

@mighty-mahi-mahi
